app/spdb/build.sbt

```diff
@@ -573,6 +574,4 @@ def gsodb(version: Version) = AppConfig(
   bundles = List(
     BundleSpec(50, "edu.gemini.smartgcal.servlet", version)
   )
-) extending List(odbproduction(version), gsodbtest_credentials(version))
-
-
+) extending List(odbproduction(version), gsodb_credentials(version))
```

Comment:
I know why you are making this change but somehow it worries me. How did it work correctly in the first place?